pyproject.toml

```diff
@@ -52,12 +103,7 @@ filterwarnings = [
 
 [tool.setuptools]
 packages = ["s2fft"]
-
-[tool.setuptools.dynamic]
-dependencies = {file = ["requirements/requirements-core.txt"]}
-optional-dependencies.docs = { file = ["requirements/requirements-docs.txt"] }
-optional-dependencies.plotting = { file = ["requirements/requirements-plotting.txt"] }
-optional-dependencies.tests = { file = ["requirements/requirements-tests.txt"] }
+write_to = "s2fft/_version.py"
```

Comment:
```suggestion
```
This is I think should belong under `tool.setuptools_scm` and it appears there is already a entry for this below